panels/info/cc-info-panel.c

```diff
@@ -1939,6 +1876,15 @@ cc_info_panel_init (CcInfoPanel *self)
       g_error_free (error);
     }
 
+  self->priv->client = udisks_client_new_sync (NULL, &error);
+
+  if (error != NULL)
+    {
+      g_critical ("Unable to get UDisks client: %s. Disk information will not be available.",
```

Comment:
I’d use `g_warning()` here just in case criticals are set to be fatal. We probably don’t want this to be fatal.